R/vroom.R

```diff
@@ -4,26 +4,116 @@ NULL
 
 #' Read a delimited file into a tibble
 #'
-#' @inheritParams readr::read_delim
-#' @param file path to a local file.
+#' @param file Either a path to a file, a connection, or literal data (either a
```

Comment:
The only changes/improvements that have been made are to the `file` path parameter here. Everything else was pulled over, as is, from readr so that `vroom()` doesn't inherit from readr anymore. When rendered, the docs are identical. IMO the scope of this PR is only to stop inheriting docs from readr (and potentially only for `vroom()` for now), not necessarily to improve the docs we have. If we find things along the way that need updating, we can make separate PRs for them.

@jennybc What other context would be helpful? Once we finish this I can start https://github.com/tidyverse/readr/issues/1421
Do you still think we should only do this `vroom()` for now or should we tackle the whole package? FWIW I've found 6 other locations where vroom inherits docs from readr.

Reply:
Note to selves: this still doesn't mention a really important feature which is that `vroom()` works with multiple files or connections. But I assume that's going to get handled with other multi-file issues that are in flight (?).